<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="walls">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>92</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>стіни</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="same_distance">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>однакова відстань</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Згенерувати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистити</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>234</width>
      <height>133</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="start_coords_label">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Координати точки старту</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="start_x_label">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>x:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="start_x">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="start_y_label">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="start_y">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="finish_coords_label">
         <property name="font">
          <font>
           <family>Fixedsys</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Координати точки фінішу</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="finish_x_label">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>x:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="finish_x">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="finish_y_label">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="finish_y">
             <property name="font">
              <font>
               <family>Fixedsys</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>414</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="grid_params_label">
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Параметри сітки</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="grid_width_label">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>ширина:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="grid_width">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="grid_height_label">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>висота:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="grid_height">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="wall_chance_label">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>стіна:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="wall_chance">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="max_dist_label">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>макс. відстань:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="max_distance">
           <property name="font">
            <font>
             <family>Fixedsys</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>15</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
